examples/risk_and_security/retail_agent/README.md

```
@@ -0,0 +1,540 @@
+  - [Table of Contents](#table-of-contents)
+  - [Key Features](#key-features)
+  - [Installation and Setup](#installation-and-setup)
+    - [Install this Workflow](#install-this-workflow)
+    - [Set Up API Keys](#set-up-api-keys)
+    - [Run the Workflow](#run-the-workflow)
+  - [Agent Capabilities](#agent-capabilities)
+  - [Example Usage](#example-usage)
+    - [Product Inquiry](#product-inquiry)
+    - [Review Submission](#review-submission)
+    - [Order Placement](#order-placement)
+    - [Order with Multiple Products](#order-with-multiple-products)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)